test/issuer/IssuerAccessFacet.test.ts

```diff
@@ -193,13 +208,29 @@ describe('IssuerAccessFacet', () => {
       const issuerAsFast = await impersonateContract(issuer, fast.address);
 
       // Add then remove Alice.
-      issuerAsFast.governorAddedToFast(alice.address);
-      issuerAsFast.governorRemovedFromFast(alice.address);
+      await issuerAsFast.governorAddedToFast(alice.address);
+      await issuerAsFast.governorRemovedFromFast(alice.address);
```

Comment:
Why did this happily pass before?

Reply:
Good question... It probably has to do with the speed at which these are executed, and we were lucky enough not to have a race condition.